gatsby-config.js

```diff
@@ -109,17 +109,17 @@ module.exports = {
         policy: [{ userAgent: "*", allow: "/" }],
       },
     },
-    {
-      resolve: `gatsby-plugin-prefetch-google-fonts`,
-      options: {
-        fonts: [
-          {
-            family: "Poppins",
-            variants: [`300`, `400`, `700`],
-          },
-        ],
-      },
-    },
+    // {
+    //   resolve: `gatsby-plugin-prefetch-google-fonts`,
+    //   options: {
+    //     fonts: [
+    //       {
+    //         family: "Poppins",
+    //         variants: [`300`, `400`, `700`],
+    //       },
+    //     ],
+    //   },
+    // },
```

Comment:
Goodbye, Poppins!
![image](https://user-images.githubusercontent.com/11503403/101362765-8858a180-386e-11eb-9b83-73cbb0f5e64f.png)

Reply:
Ah yeah I'm just going to import this manually instead of using this plugin. It has a habit of causing broken builds and would prefer to avoid that situation 😂